com.termius.termius.appdata.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- © 2019 Termius Corporation.> -->
+<component>
+ <id>com.termius.termius</id>
+ <metadata_license>CC0-1.0</metadata_license>
+ <project_license>MIT</project_license>
```

Comment:
Is this a left-over? The app appears to be propritary freemium.

Reply:
Yes but what is licence accepted have make non-free and with that flatpak no build

Reply:
https://github.com/flathub/com.slack.Slack/blob/master/com.slack.Slack.appdata.xml#L5 is an example.

Reply:
thank